### DISCUSSION TITLE
Duplicate a template prompt to start a new prompt with all the boilerplate prepared

### DISCUSSION THREAD

**Discussion by jakelucasnyc** [langfuse/langfuse#4012] [Ideas]
### Describe the feature or potential improvement

Depending on the model I'm prompting, there's often some boilerplate and common inputs I always use, so I'd love to be able to create a template prompt with that boilerplate already there and be able to duplicate that template as a starting point for a new specific prompt for a node in my AI pipeline.

**Comment by marcklingen** [comment]
This is great, thanks for sharing this. This was added to datasets and is really popular for these kinds of workflows.

**Comment by marcklingen** [comment]
Forgot to update this thread, this was added by @marliessophie, thanks again for raising this!
